xonsh/xontribs.py

```diff
@@ -109,18 +109,25 @@ def xontrib_metadata():
 def xontribs_load(names, verbose=False):
     """Load xontribs from a list of names"""
     ctx = builtins.__xonsh__.ctx
+    res = 0
```

Comment:
Why 0/1/2?

Can we name this something more descriptive? Or better, document what this function returns and why?

Reply:
Is it better now?

Reply:
Oh! That's why more than I was expecting (I was thinking, like, docstring), but yes, that works.